django-stubs/db/models/fields/__init__.pyi

```diff
@@ -69,6 +69,10 @@ class Field(RegisterLookupMixin, Generic[_ST, _GT]):
     def formfield(self, **kwargs) -> FormField: ...
     def contribute_to_class(self, cls: Type[Model], name: str, private_only: bool = ...) -> None: ...
     def to_python(self, value: Any) -> Any: ...
+    def get_attname(self) -> str: ...
+    def value_from_object(self, obj) -> _GT: ...
```

Comment:
Could you extract this line (whuch causes test failures) in a different PR? It's probably not obvious of a change, and requires a test.